packages/subiquity_client/lib/src/types.dart

```diff
@@ -196,14 +196,29 @@ enum ProbeStatus { PROBING, FAILED, DONE }
 
 enum Bootloader { NONE, BIOS, UEFI, PREP }
 
+@freezed
+class OsProber with _$OsProber {
+  @JsonSerializable(fieldRename: FieldRename.snake)
+  const factory OsProber({
```

Comment:
P.S. Missing coverage for this const constructor is a bug in Dart: https://github.com/flutter/flutter/issues/95008 (already fixed in the latest Flutter master)